src/Swarm/Game/Display.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
I had to remove these from the global defaults and add them manually to individual modules, because the `BoolExpr.hs` module does not work with this extension.

Reply:
Could you use `NoDerive` in boolexpr?